packages/core/src/layout/Sidebar/Settings/AuthProviderList.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import { Divider } from '@material-ui/core';
+
+type Props = {
+  providerSettings?: React.ReactNode;
+};
+
+export const AuthProvidersList = ({ providerSettings }: Props) => {
+  if (!providerSettings) {
```

Comment:
this decision feels out of place here - especially since this one draws a divider further down as well

i expect the caller to decide that if there are any provider settings, then draw a divider and this component - ie, the wrapper is the one that knows what the layout and dividers and stuff is, not this one